cloud-gallery/body/package.json

```diff
@@ -7,6 +7,8 @@
 		"build.client": "vite build",
 		"build.ssr": "vite build --ssr src/entry.ssr.tsx",
 		"lint": "eslint \"src/**/*.ts*\"",
+		"check-format": "prettier --check --ignore-path=../.prettierignore .",
+		"format": "prettier --write --ignore-path=../.prettierignore .",
```

Comment:
is the `--ignore-path` part necessary? 😮

Reply:
Yes because we are running prettier in the `cloud-gallery/body` (for example) directory and by default prettier will only look for `./.prettierigore` (i.e. in the local directory).